nuxt.config.js

```diff
@@ -2,6 +2,20 @@ require("dotenv").config();
 const pkg = require("./package");
 const axios = require('axios');
 
+const env = {
+  BASE_URL: process.env.BASE_URL || "http://localhost:3000",
+  PORT: process.env.WEB_PORT || "3000",
```

Comment:
Why does BASE_URL have a port of we're declaring port here separately? Do we need both?